test/TravelAgent-test.js

```diff
@@ -0,0 +1,77 @@
+import chai, { expect } from 'chai';
+import User from '../src/User';
+import Traveler from '../src/Traveler';
+import TravelAgent from '../src/Travel-agent'
+import travelerData from '../data/sampleTravelerData';
+import tripsData from '../data/sampleTripsData';
+import destinationData from '../data/sampledestinationData';
+
+describe('TravelAgent', function() {
+  let travelAgent;
+
+  beforeEach(function() {
+    travelAgent = new TravelAgent();
+  });
+
+  it('should be a function', function() {
+    expect(TravelAgent).to.be.a('function');
+  });
+
+  it('should be an instance of travelAgent', function() {
+    expect(travelAgent).to.be.an.instanceof(TravelAgent);
+  });
+
+  it('should be able to find a traveler\'s id by their name', function() {
+    expect(travelAgent.findUser('Rachael Vaughten', travelerData)).to.equal(2)
+  })
+
+  it('should have an id property of the traveler they are searching for', function() {
```

Comment:
Nice work testing these, when moving to fetch data from the remote server. You will want to test the sad paths of these if a user doesn't exist.

Reply:
Agreed, thanks for the reminder!